tools/travis.sh

```diff
@@ -22,6 +23,13 @@ update_distro_packages()
     case "${DISTRO}" in
         centos*)
             sudo yum install -y epel-release
+            # remove fake aspcud and install our own rpms
+            wget https://github.com/xapi-project/xapi-travis-scripts/raw/master/aspcud/aspcud-1.9.0-1.el7.centos.x86_64.rpm
+            wget https://github.com/xapi-project/xapi-travis-scripts/raw/master/aspcud/clasp-3.1.0-1.el7.centos.x86_64.rpm
+            wget https://github.com/xapi-project/xapi-travis-scripts/raw/master/aspcud/gringo-4.4.0-1.el7.centos.x86_64.rpm
+            sudo rm -rf /usr/bin/aspcud
+            sudo yum install -y *.rpm
+            rm *.rpm
```

Comment:
We have to fix up aspcud for the `check_unused` job as well - maybe we could move this into its own function, and run it from `update_distro_packages` and the `check_unused_branch`, so we don't do more work than we need to.

Reply:
Ok